axon-server-connector/src/main/java/org/axonframework/axonserver/connector/query/AxonServerQueryBus.java

```diff
@@ -129,12 +129,12 @@ public class AxonServerQueryBus implements QueryBus, Distributed<QueryBus> {
 
     private static final int DIRECT_QUERY_NUMBER_OF_RESULTS = 1;
     private static final long DIRECT_QUERY_TIMEOUT_MS = TimeUnit.HOURS.toMillis(1);
-    private static final int SCATTER_GATHER_NUMBER_OF_RESULTS = -1;
 
     private static final int QUERY_QUEUE_CAPACITY = 1000;
 
     private final AxonServerConnectionManager axonServerConnectionManager;
     private final AxonServerConfiguration configuration;
+    @SuppressWarnings("unused")
```

Comment:
I am skipping this file as it'll be removed/replaced as part of https://github.com/AxonFramework/AxonFramework/issues/3488.